CHANGES_NEXT_RELEASE

```diff
@@ -27,3 +27,5 @@ Add:  Functional test to PUT/POST/PATCH with 'empty' JSON object as payload (Iss
 Fix:  performance problem in NGSIv2 API due to uncontrolled log trace printing
 Add:  empty attribute values supported in NGSIv1 API operations (Issue #1358)
 Add:  Add CLI argument for notification mode (Issue #1370)
+Fix:  Recovering $where from 0.23.0 code bases to search for subscriptions (-noCache flag)
```

Comment:
Taking care in PR https://github.com/telefonicaid/fiware-orion/pull/1410

NTC (in this PR).